javascriptv3/example_code/dynamodb/README.md

```diff
@@ -15,13 +17,15 @@ Amazon DynamoDB is a fully managed NoSQL database service that provides fast and
 - [Additional resources](#-A)
 
 ## ⚠️ Important information
-* Running the tests might result in charges to your AWS account.
-* We recommend that you grant your code the least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
-* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
```

Comment:
Insert first line of "Important" list [from template](https://github.com/awsdocs/aws-doc-sdk-examples/wiki/Service-level-README-template): 
- * Running this code might result in charges to your AWS account.